File: app/src/main/java/com/jimzrt/umsmounter/model/DownloadItem.java
package com.jimzrt.umsmounter.model;

import androidx.annotation.NonNull;


public class DownloadItem implements Comparable<DownloadItem> {
    public String id;
    public String name;
    public String[] tags;
    public String url;
    public Release[] releases;


    @Override
    public int compareTo(@NonNull DownloadItem o) {
        return name.compareTo(o.name);
    }
}

File: app/src/main/java/com/jimzrt/umsmounter/model/DownloadItem.kt
package com.jimzrt.umsmounter.model

class DownloadItem : Comparable<DownloadItem> {
    var id: String? = null

    @JvmField
    var name: String? = null
    lateinit var tags: Array<String>

    @JvmField
    var url: String? = null
    lateinit var releases: Array<Release>
    override fun compareTo(o: DownloadItem): Int {
        return name!!.compareTo(o.name!!)
    }
}